.swiftlint.yml

```diff
@@ -0,0 +1,117 @@
+#
+# SwiftLint configuration file commonly used in STRV (v 0.1.2)
+#
+# Feel free to send pull request or suggest improvements!
+#
+
+
+#
+# Rule identifiers to exclude from running.
+#
+disabled_rules:
```

Comment:
why is this rule disabled?

Reply:
Trailing_whitespace rule removed from disables section in .swiftlint.yaml

Reply:
Trailing_whitespace rule removed from disables section in .swiftlint.yaml